src/lib/ebus/symbol.h

```diff
@@ -123,6 +123,12 @@ class SymbolString {
    */
   explicit SymbolString(bool isMaster = false) { m_isMaster = isMaster; }
 
+  /**
+   * Copy assignment operator.
+   * @param other the @a SymbolString to copy from.
+   */
+  SymbolString& operator=(const SymbolString& other) = default;
+
```

Comment:
please remove this one, it was addressed with 058c7d6 already

Reply:
Done.